gateway-util-common/pom.xml

```diff
@@ -6,9 +6,7 @@
   The ASF licenses this file to You under the Apache License, Version 2.0
```

Comment:
This file is just formatting changes. Please remove the changes.

Reply:
removed changes